glib/src/subclass/types.rs

```diff
@@ -19,6 +19,7 @@ pub struct InitializingType<T>(pub(crate) Type, pub(crate) marker::PhantomData<*
 
 impl<T: ObjectSubclass> InitializingType<T> {
     /// Adds an interface implementation for `I` to the type.
+    #[doc(alias = "g_type_add_interface_static")]
```

Comment:
@sdroege regarding this one, does it make sense to have doc aliases for subclassing support as well? :)

Reply:
Sure why not

Reply:
I will do another round in a separate MR later then :)